intermediate_html_css/intermediate_css_concepts/more_text_styles.md

```diff
@@ -27,16 +27,17 @@ body {
 
 The point of this somewhat ridiculous string of font-families is to try using the default font of the system's user interface. It will go through each of those fonts until it finds one that is installed on the system, and then use that. Using a stack like this often produces pleasing results, especially if you're going for a somewhat 'neutral' font style.
 
-#### Online font libraries
+#### Web fonts
 
-One popular and easy method to get fonts that are *not* installed on a user's computer is to use the online [Font Library](https://fontlibrary.org/), or the premium, but non-free, [Adobe Fonts](https://fonts.adobe.com/).
+If you want to use a font that will not be available on the user's device you will need to import the font from an online source, either a font library or an asset on your site. Either method will import that font and make it available for you to use in your CSS `font-family` properties.
 
-<div class="lesson-note lesson-note--warning" markdown="1">
-### A warning about Google fonts
+Keep in mind that it's important to add a fallback font.  If you're linking to an external API, you have no guarantee that the URL won't change, or that the external API won't go down at some point. Having a reasonable fallback means that if something goes wrong, at least your site won't look completely broken.
 
-As you look into online font libraries to use you may come across [Google Fonts](https://fonts.google.com/). We strongly advise against using them, as websites that use Google Fonts [violate the GDPR](https://www.cookieyes.com/documentation/features/integrations/google-fonts-and-gdpr/). Do not follow the instructions in that article on how to make Google fonts compliant, as we will be going over downloading fonts in this lesson.
+There are performance issues with using imported files, which you will read more about in the assignments. Ultimately it is best to use a font stack instead of imported fonts if your design allows for it, however, there are times you will want to use an imported font.
```

Comment:
```suggestion
There are performance issues with using imported files, which you will read more about in the assignments. Therefore, it is best to use a font stack instead of imported fonts whenever your design allows for it. However, there are times you will want to use an imported font.
```